lua/neoclip/settings.lua

```diff
@@ -42,6 +42,7 @@ local settings = {
             paste = 'ctrl-p',
             paste_behind = 'ctrl-k',
             custom = {},
+            opts = {},
```

Comment:
A bit weird if this lives under `keys` maybe, could we have a single prompt entry could be used at the top-level for both telescope and fzf-lua?